Corona-Warn-App/src/main/res/xml/network_security_config.xml

```diff
@@ -8,6 +8,8 @@
             expiration="2024-02-12"
             tools:ignore="MissingBackupPin">
```

Comment:
Can this be removed now that there are more pins?

Reply:
done